frontend/src/views/flight-details.test.js

```diff
@@ -0,0 +1,10 @@
+// src/components/channel-list.test.js
+import React from "react";
+import { render } from "@testing-library/react";
+
+import FlightDetails from "./flight-details";
+
+test("FlightDetails component", () => {
+  const { asFragment } = render(<FlightDetails />);
+  expect(asFragment()).toMatchSnapshot();
+});
```

Comment:
I suggest testing the button you have in your component as well. As for now, that button does nothing, but as soon as you give it a functionality, then test it.

Reply:
Yes, I will do it. Thanks! :)